gen/rules/src/main/scala/clue/gen/GraphQLGen.scala

```diff
@@ -8,6 +8,7 @@ import scala.meta._
 import cats.syntax.all._
 import metaconfig.Configured
 import cats.effect.IO
+import cats.effect.unsafe.implicits.global
```

Comment:
This is executed as part of the scalafix, so we want to use the unsafe calls

Reply:
I see. This provides an implicit `IORuntime` :+1: